include/irr/asset/filters/dithering/CPrecomputedDither.h

```diff
@@ -26,30 +26,74 @@ namespace irr
 				/*
 					The state requires only input dithering image
 					which buffer will be used in dithering process
-					in full extent of image.
+					in full extent of image. Dithering will be applied
+					using first 0th mipmap and layer either.
 				*/
 				
 				class CState : public CDither::CState
 				{
 					public:
 						CState(const asset::ICPUImage* const ditheringImage) 
```

Comment:
maybe take an `ICPUImageView` then one has a choice (indirectly) of the mip-map and layer that could be used for the dither.

Reply:
Okay